third_party/libgtest/tnt/CMakeLists.txt

```diff
@@ -13,6 +13,12 @@ set(SRCS
     ${SRC_DIR}/gtest-all.cc
 )
 
+if (MSVC)
+  # The /Zc:__cplusplus compiler option enables the __cplusplus preprocessor macro to 
```

Comment:
See my previous comment about this flag.

Reply:
@romainguy, I don't have a sandox with all the changes in right now to verify that everything builds fine with msvc when moving this option to the top makefile.

Can we approve this PR as is, and in the next PR I'll submit all my remaining changes, plus the move of the /Zc:__cplusplus option to the root cmake (where I agree it should be).